scripts/us_ca_chhs/healthcare_facility_bed_counts/preprocess_csv.py

```diff
@@ -0,0 +1,261 @@
+               'Count_HospitalBed_AcuteRespiratoryCareBed',
+               'Count_HospitalBed_AcutePsychiatricCareBed',
+               'Count_HospitalBed_ChemicalDependencyRecoveryBed',
+               'Count_HospitalBed_PediatricIntensiveCareUnitBed',
+               'Count_HospitalBed_LaborAndDeliveryBed',
+               'Count_HospitalBed_IntermediateCareDevelopmentallyDisabledBed',
+               'Count_HospitalBed_PsychiatricHealthBed',
+               'Count_HospitalBed_PediatricDayAndRespiteCareBed',
+               'Count_HospitalBed_DialysisStationsBed',
+               'Count_HospitalBed_CorrectionalTreatmentCenterBed']
+
+new_df = pd.DataFrame(columns=new_columns)
+
+FDR_to_type = {
+    'SKILLED NURSING FACILITY': 'SkilledNursingFacility',
+    'INTERMEDIATE CARE FACILITY-DD/H/N/CN/IID': 'IntermediateCareFacility',
+    'CONGREGATE LIVING HEALTH FACILITY': 'CongregateLivingHealthFacility',
+    'INTERMEDIATE CARE FACILITY': 'IntermediateCareFacility',
+    'HOSPICE FACILITY': 'HospiceFacility',
+    'GENERAL ACUTE CARE HOSPITAL': 'GeneralAcuteCareFacility',
+    'ACUTE PSYCHIATRIC HOSPITAL': 'AcutePsychiatricFacility',
+    'HOSPICE': 'HospiceFacility',
+    'PEDIATRIC DAY HEALTH & RESPITE CARE FACILITY': 'PediatricDayHealthAndRespiteCareFacility',
+    'CHRONIC DIALYSIS CLINIC': 'ChronicDialysisFacility',
+    'CHEMICAL DEPENDENCY RECOVERY HOSPITAL': 'ChemicalDependencyRecoveryFacility',
+    'CORRECTIONAL TREATMENT CENTER': 'CorrectionalTreatmentCenterFacility'
+}
+
+bed_type_dict = {
+    'SKILLED NURSING': 'SkilledNursingBed',
+    'SPECIAL TREATMENT PROGRAM': 'SpecialTreatmentProgramBed',
+    'INTERMEDIATE CARE/DD HABILITATIVE': 'IntermediateCareHabilitativeBed',
+    'INTERMEDIATE CARE/DD NURSING': 'IntermediateCareNursingBed',
+    'CONGREGATE LIVING HEALTH FACILITY': 'CongregateLivingHealthFacilityBed',
+    'INTERMEDIATE CARE': 'IntermediateCareBed',
+    'HOSPICE': 'HospiceBed',
+    'CORONARY CARE': 'CoronaryCareBed',
+    'INTENSIVE CARE': 'IntensiveCareBed',
+    'INTENSIVE CARE NEWBORN NURSERY': 'IntensiveCareNewbornNurseryBed',
+    'PERINATAL': 'PerinatalBed',
+    'UNSPECIFIED GENERAL ACUTE CARE': 'UnspecifiedGeneralAcuteCareBed',
+    'BURN': 'BurnBed',
+    'PEDIATRIC': 'PediatricBed',
+    'RENAL TRANSPLANT': 'RenalTransplantBed',
+    'REHABILITATION': 'RehabilitationBed',
+    'ACUTE RESPIRATORY CARE': 'AcuteRespiratoryCareBed',
+    'ACUTE PSYCHIATRIC CARE': 'AcutePsychiatricCareBed',
+    'CHEMICAL DEPENDENCY RECOVERY': 'ChemicalDependencyRecoveryBed',
+    'PEDIATRIC INTENSIVE CARE UNIT': 'PediatricIntensiveCareUnitBed',
+    'LABOR AND DELIVERY': 'LaborAndDeliveryBed',
+    'INTERMEDIATE CARE/DD': 'IntermediateCareDevelopmentallyDisabledBed',
+    'PSYCHIATRIC HEALTH': 'PsychiatricHealthBed',
+    'PEDI. DAY & RESPITE CARE': 'PediatricDayAndRespiteCareBed',
+    'DIALYSIS STATIONS': 'DialysisStationsBed',
+    'CORRECTIONAL TREATMENT CENTER': 'CorrectionalTreatmentCenterBed',
+}
```

Comment:
I wonder if handling a few special cases, and then just doing a Title Case + strip whitespace would be better than this long map. I will leave this comment for other reviewers to see what they think.